packages/in_app_purchase/example/lib/main.dart

```diff
@@ -9,6 +9,7 @@ import 'package:in_app_purchase/in_app_purchase.dart';
 import 'consumable_store.dart';
 
 void main() {
+  InAppPurchaseConnection.enablePendingPurchases();
```

Comment:
nit: Comment here explaining what this is and linking to the Play documentation would be useful.